src/App.js

```diff
@@ -1,41 +1,34 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Navbar, RequiresAuth } from "./components/index";
+import Mockman from "mockman-js";
+import { ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import { Routes , Route } from 'react-router-dom';
+import { HomePage , SignIn , SignUp , History, Likes, WatchLater, PlayList, SingleVideo} from "./Pages/index";
```

Comment:
we don't need to mention the index at the end, 

`from "./Pages` 
this would also work